process/current_drive.py

```diff
@@ -721,13 +880,27 @@ def cudriv(self, output: bool):
                 "(gamma_ecrh)",
                 current_drive_variables.gamma_ecrh,
             )
-        elif current_drive_variables.iefrf == 12:
+        if current_drive_variables.iefrf == 12:
             po.ovarre(
                 self.outfile,
                 "EBW plasma heating efficiency",
                 "(xi_ebw)",
                 current_drive_variables.xi_ebw,
             )
+        if current_drive_variables.iefrf in [12, 13]:
+            po.ovarre(
+                self.outfile,
+                "EC harmonic number",
+                "(harnum)",
+                current_drive_variables.harnum,
+            )
+        if current_drive_variables.iefrf == 13:
```

Comment:
Would more 'correct' to move this block under the `elif current_drive_variables.iefrf == 12:` and have it as `elif current_drive_variables.iefrf == 13:` and then have the if in block below as is.

Reply:
It doesn't really matter in the order of the output. At the moment the output goes efficiency scaling value -> harmonic number -> wave mode. `iefrf = 13` dose not a efficiency scaling value.